app/src/main/java/com/duckduckgo/app/statistics/VariantManager.kt

```diff
@@ -41,9 +42,15 @@ interface VariantManager {
             Variant(key = "sc", weight = 0.0, features = emptyList()),
             Variant(key = "se", weight = 0.0, features = emptyList()),
 
+            // All groups in an experiment (control and variants) MUST use the same filters
             Variant(key = "mw", weight = 1.0, features = emptyList()),
             Variant(key = "mx", weight = 1.0, features = listOf(VariantFeature.TabSwitcherGrid))
         )
+
+        private fun isEnglishLocale(): Boolean {
+            val locale = Locale.getDefault()
+            return locale != null && locale.language == "en"
```

Comment:
Can we use the `DeviceInfo.getLanguage()` method here?

Reply:
Doing this will complicate things a bit since it needs injecting and is currently needed in an interface where we can't really inject.